kogni/erste_woche/einmaleins.tex

```diff
@@ -48,6 +48,8 @@
 
 \item[Mensa:] Die Mensen sind der Ort in dem man sich zum (un-)regelmäßigen Mittagessen trifft. Jede Mensa hat einen Speiseplan, welchen man auf der Seite des Studierendenwerks einsehen kann\footnote{\url{http://www.my-stuwe.de/mensa/}}.	%TODO insert \link{}{}?
 
+\item[Memes] Eine Memeseite, bei der jeder mitmachen kann ist auf Instagram unter kogni_memes zu finden
```

Comment:
Please add colon after `Memes`.

Also use `\href{https://www.instagram.com/kogni_memes/}{\faInstagram @kogni\_memes}` to get a more nice render.

Reply:
okay. btw idk if FSK supports this but they should. In any case i can make a meme out of this

Reply:
@PhictionalOne  we will discuss this at our session and I will come back here